src/fuck/Models/Command.cs

```diff
@@ -1,17 +1,18 @@
 ﻿using System.Collections.Generic;
-using System.Linq;
 
 namespace fuck.Models
 {
     public class Command
     {
-        public string OriginalCommand { get; set; }
-        public IEnumerable<string> UpdatedCommand => Recommendations.Select(x => Program + " " + x + " " + Arguments);

```

Comment:
Yup!  Checkout out the section "Expression Bodied Functions and Properties": https://msdn.microsoft.com/en-us/magazine/dn802602.aspx